sirepo/raydata/analysis_driver/__init__.py

```diff
@@ -78,6 +88,22 @@ def get_run_log(self):
     def has_analysis_pdfs(self):
         return len(self.get_analysis_pdf_paths()) > 0
 
+    def render_papermill_script(self, input_f, output_f):
+        pkjinja.render_resource(
+            _PAPERMILL_SCRIPT,
+            PKDict(
+                input_f=input_f,
+                output_f=output_f,
+                papermill_args=" ".join(self.get_papermill_args()),
+                conda_prefix=_cfg.conda_prefix,
+                conda_env=self.get_conda_env(),
+                catalog_name=self.catalog_name,
+            ),
+            output=self.get_papermill_script_path(),
```

Comment:
Store self.get_papermill_script_path a local var, use here, and return at the end. It helps alert the reader that where the file is written to is the path we return.

Reply:
a4175ab0127a5b2207443ee18cab9d384c401eb9